core/modules/ckeditor/ckeditor.module

```diff
@@ -4,7 +4,7 @@
  * Provides integration with the CKEditor WYSIWYG editor.
  */
 
-define('CKEDITOR_VERSION', '4.7.3+2018-04-18-security-patch');
+define('CKEDITOR_VERSION', '4.10.1');
```

Comment:
Looks like this should be updated to 4.11.0

Reply:
4.11.1 is out actually ...and changes were only limited to `CHANGES.md`